pallets/committee/src/lib.rs

```diff
@@ -359,23 +382,30 @@ pub mod pallet {
         }
     }
 
+    /// Initialize council members. Can only be done once.
+    /// Constituent members must be initialized later by voting by the council
     impl<T: Config> InitializeMembers<AccountIdFor<T>> for Pallet<T> {
         fn initialize_members(members: &[AccountIdFor<T>]) {
             if !members.is_empty() {
                 assert!(
-                    Members::<T>::get().is_empty(),
+                    Members::<T>::iter().count() == 0,
```

Comment:
Just spotted this simplification
```suggestion
                    assert_eq!(Members::<T>::iter().count(), 0, "Members are already initialized!");
```